src/content/api/cli.md

```diff
@@ -244,6 +244,52 @@ Specify a different [configuration](/configuration) file other than `webpack.con
 npx webpack --config example.config.js
 ```
 
+In case your configuration file exports multiple configurations you can use `--config-name` to specify the name of configuration to use.
```

Comment:
```suggestion
In case your configuration file exports multiple configurations, you can use `--config-name` to specify which configuration to run.
```